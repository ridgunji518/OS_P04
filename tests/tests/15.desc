Freeing pages when process ends
