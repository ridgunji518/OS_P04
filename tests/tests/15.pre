./edit-makefile.sh ../solution/Makefile test_15_a,test_15_b > ../solution/Makefile.test
cp -f tests/test_15_a.c ../solution/test_15_a.c
cp -f tests/test_15_b.c ../solution/test_15_b.c
cd ../solution/
make -f Makefile.test clean
make -f Makefile.test xv6.img
make -f Makefile.test fs.img
cd ../tests

