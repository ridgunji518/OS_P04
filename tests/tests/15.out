XV6_TEST_OUTPUT Max, 256MB - program base pages:0 huge pages:65
XV6_TEST_OUTPUT Max, 256MB - program base pages:0 huge pages:65
